dao/OrderRepository.go

```diff
@@ -271,3 +271,123 @@ func (r *OrderPostgres) GetServices(in *emptypb.Empty) (*courierProto.ServicesRe
 	}
 	return &Services, nil
 }
+
+func (r *OrderPostgres) GetAllCompletedOrdersOfCourierServiceFromDB(limit, page, idService int) ([]Order, int) {
```

Comment:
Similar comment about methods to access the DB. There is too much repetition right now, especially given that you need to parse the data in the same data structures. It is still ok to have separate queries for different search criteria, just try to unify at least the code for parsing and so on.

Reply:
Btw, getting all completed orders sounds not right. Courier would have thousand of them over time. That will make your API and FE respond slowly or break at all.